repair/repair.cc

```diff
@@ -1010,7 +1010,7 @@ void repair::shard_repair_task_impl::release_resources() noexcept {
     ignore_nodes = {};
     neighbors = {};
     dropped_tables = {};
-    nodes_down = {};
+    _nodes_down = {};
```

Comment:
Why not include `ignore_nodes`, `neighbours` and `dropped_tablets` into this set too?

Reply:
Oy, I see there's a bunch of members with no leading underscore in `shard_repair_task_impl`.
They are public members inherited from `repair_info` in b4ad914fe1f, and as such I'm not sure what should be the naming convention for them.
We definitely want the underscore for private members, and we don't requires it for struct public members, but I'm not sure about the policy for class that have both public and private members.

Reply:
We are not that strict on public members, in some cases they have leading underscore, in other cases they don't.
We are more strict on requiring leading underscore for private members. I this PR can be dropped.

Reply:
> We are not that strict on public members, in some cases they have leading underscore, in other cases they don't. We are more strict on requiring leading underscore for private members. I this PR can be dropped.

I'd leave only the patch for _topo_sm then.

Reply:
Ok, sounds good.

Reply:
done